src/mono_font/mono_text_style.rs

```diff
@@ -385,7 +385,7 @@ pub struct MonoTextStyleBuilder<'a, C> {
     style: MonoTextStyle<'a, C>,
 }
 
-impl<C> MonoTextStyleBuilder<'_, C> {
+impl<'a, C: PixelColor> MonoTextStyleBuilder<'a, C> {
```

Comment:
Nit: could you please change this to:

```suggestion
impl<'a, C> MonoTextStyleBuilder<'a, C> where C: PixelColor {
```

to be consistent with the rest of the codebase.